R/r2dii.analysis-package.R

```diff
@@ -0,0 +1,8 @@
+#' @keywords internal
+"_PACKAGE"
+
+# The following block is used by usethis to automatically manage
+# roxygen namespace tags. Modify with care!
+## usethis namespace: start
+## usethis namespace: end
+NULL
```

Comment:
LGTM, but what's the use of this?

Reply:
Roxygen2 uses this file to automatically generate the helpfile you see when you run `?r2dii.analysis`. In turn, you generate this file with `usethis::use_package_doc()`.

When I hear of a cool new package and have no idea 
![image](https://user-images.githubusercontent.com/5856545/113868903-04072400-9765-11eb-800d-cea22c5784c1.png)

--

At the very bottom, that  helpfile has a convenient link to the entire index, which is a great way to see all features at a glance:

![image](https://user-images.githubusercontent.com/5856545/113869105-3153d200-9765-11eb-9bb4-2e7adff2d618.png)